src/samplers/string.js

```diff
@@ -96,8 +101,8 @@ function relativeJsonPointerSample() {
   return '1/relative/json/pointer';
 }
 
-function regexSample() {
-  return '/regex/';
+function regexSample(min, max, _propertyName, pattern) {
+  return pattern ? fromRegExp(pattern) : truncateString('/regex/', min, max)
```

Comment:
we don't need to adjust it. Let's leave it is.
This is a string with `format: regex` which means it has to be a valid regex. I doubt someone will provide regex for regex.

Reply:
Ah, I misunderstood what `format: regex` meant. Thank you, fixed